tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/Steps/OrderStatusSteps.cs

```diff
@@ -1,10 +1,17 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
 using System.Threading.Tasks;
 using FluentAssertions;
+using Microsoft.VisualBasic.FileIO;
+using NHSD.BuyingCatalogue.EmailClient.IntegrationTesting.Drivers;
+using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Support;
 using NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.Utils;
 using NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.Data;
 using NHSD.BuyingCatalouge.Ordering.Api.Testing.Data.Entities;
 using TechTalk.SpecFlow;
+using static System.String;
```

Comment:
I think this type of syntax was frowned upon in one of the coding standard meetings. Do you need to use a static using statement here?